src/LoginServer/Network/LoginPacketHandler.cs

```diff
@@ -277,13 +280,13 @@ public void CB_COMMANDER_CREATE(LoginConnection conn, Packet packet)
 		[PacketHandler(Op.CB_COMMANDER_DESTROY)]
 		public void CB_COMMANDER_DESTROY(LoginConnection conn, Packet packet)
 		{
-			var index = packet.GetByte();
+			var id = packet.GetLong();
```

Comment:
They finally decided to use the ids instead of the indexes? Hallelujah.

Reply:
I hate those indexes. I guess if they seriously needed a smaller packet size they can do it, but really?

Reply:
Yea, no idea what the idea behind those indexes was.